ui/app/components/navigate-input.js

```diff
@@ -61,8 +61,7 @@ export default Component.extend(FocusOnInsertMixin, {
     return `cert/${key}`;
   },
   onEnter: function(val) {
-    let baseKey = this.get('baseKey');
-    let mode = this.get('mode');
+    let { baseKey, mode } = this;
```

Comment:
So jealous of your native getters.

Reply:
When I realized we could do destructuring like this now, it was a bit of a 🤯.